### PR TITLE
Apply redesign to nav and footer

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Update to latest global nav version
- Apply redesign to nav and footer as per Jira ticket 

## QA

- View the site locally in your web browser at: https://microk8s-io-614.demos.haus/
- Check nav and footer against design
- Make sure all links work as expected **(/features will 404, this will be fixed once https://github.com/canonical/microk8s.io/pull/613 is merged)**

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-5794

### CODE REVIEW

File: templates/partial/_navigation.html [hunk 1]
```diff
@@ -1,39 +1,44 @@
...
       <a href="#navigation-closed" class="p-navigation__toggle--close" title="close menu">Close menu</a>
     </div>
 
-    <nav class="p-navigation__nav" aria-label="Main">
-      <ul class="p-navigation__items">
-        <li class="p-navigation__item {% if request.path in ['/high-availability',] %}is-selected{% endif %}">
-          <a class="p-navigation__link" href="/high-availability">Features</a>
+    <nav class="p-navigation__nav col-9" aria-label="main-navigation">
+      <ul class="p-navigation__items" style="margin-left: -.75rem;">
+        <li class="p-navigation__item">
```
Comment (anthonydillon) [inline templates/partial/_navigation.html:21]: Looks like these links are missing the `is-selected` if the `request.path` matches.

**Review by anthonydillon — CHANGES_REQUESTED** [review]
LGTM one small comment about the `is-selected` as this makes it difficult to know where you are.

**Comment by mtruj013** [comment]
Thanks @anthonydillon, fixed! Could you take a look again?

**Comment by anthonydillon** [comment]
Thanks for the fixes @mtruj013 Looks and works great!

**Review by anthonydillon — APPROVED** [review]

**Comment by lyubomir-popov** [comment]
u-no-margin--bottom here please:
image
(driveby)


logo too indented here:
image


menu items should align under the wordmark microk8s (coukld be a vanilla bug)


would be nice to retain some layout on medium screens, e.g.:
image

**Comment by mtruj013** [comment]
@lyubomir-popov ready for another look

**Comment by lyubomir-popov** [comment]
LGTM, thanks @mtruj013 . can I as kfor a driveby - add u-no-margin-bottom on the landing page h1

**Comment by mtruj013** [comment]
@lyubomir-popov sorry, totally thought I'd added it. It's there now

**Comment by juanruitina** [comment]
Lovely to see the new navigation and footer. We need however to implement the [sliding navigation](https://vanillaframework.io/docs/patterns/navigation#sliding) here so the global nav shows up as it should on mobile: as only one item you can then open. You can see what I mean [here on Figma](https://www.figma.com/file/BSP7FqnRMcMvO7kCozJKUa/Ubuntu.com-navigation-update-Iteration-9?type=design&node-id=1323%3A3714&mode=design&t=8EWcyAlY63UIVgJz-1). @petesfrench might be able to help with the implementation, he is familiar with it.

**Comment by petesfrench** [comment]
> Lovely to see the new navigation and footer. We need however to implement the [sliding navigation](https://vanillaframework.io/docs/patterns/navigation#sliding) here so the global nav shows up as it should on mobile: as only one item you can then open. You can see what I mean [here on Figma](https://www.figma.com/file/BSP7FqnRMcMvO7kCozJKUa/Ubuntu.com-navigation-update-Iteration-9?type=design&node-id=1323%3A3714&mode=design&t=8EWcyAlY63UIVgJz-1). @petesfrench might be able to help with the implementation, he is familiar with it.

Happy to help with this, but the best course of action would be to implement it on the global-nav itself. This needs to be done at one point and will mean it is available across all projects.

**Comment by mtruj013** [comment]
re: sliding navigation in the global nav. It was decided during stand-up the this will be updated directly on the global-nav, we will go live with this site as is

@juanruitina would you mind giving UX+1 if all else is well?

**Pull request merged**